### PR TITLE
update ubtue_Sciendo.js

### PR DESCRIPTION
**Pull request by socheres**

### CODE REVIEW

File: ubtue_Sciendo.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"translatorID": "bf64f8a7-89b4-4a79-ae80-70630e428f35",
 	"label": "Sciendo",
-	"creator": "Madeesh Kannan",
+	"creator": "Madeesh Kannan", "Timotheus Kim",
```
Comment (mtrojan-ub) [inline ubtue_Sciendo.js:4]: Are you sure this syntax works? seems to be invalid JSON to me.

File: ubtue_Sciendo.js [hunk 2]
```diff
@@ -44,18 +44,22 @@ function detectWeb(doc, url) {
...
-		let title = ZU.trimInternal(rows[i].textContent);
-		if (!href || !title) continue;
-		found = true;
-		items[href] = title;
-	}
-	return found ? items : false;
+function getSearchResults(doc, checkOnly) {
+  var items = {};
+  var found = false;
+  // TODO: adjust the CSS selector
```
Comment (mtrojan-ub) [inline ubtue_Sciendo.js:50]: What about all the open TODO comments?

File: ubtue_Sciendo.js (hunk 1 above)
Comment (socheres) [reply ubtue_Sciendo.js:4]: Indeed it is invalid. It should be "Madeesh Kannan and Timotheus Kim"

File: ubtue_Sciendo.js (hunk 2 above)
Comment (socheres) [reply ubtue_Sciendo.js:50]: This can be removed.

File: SageJournals.js [hunk 3]
```diff
@@ -0,0 +1,389 @@
+{
+	"translatorID": "908c1ca2-59b6-4ad8-b026-709b7b927bda",
```
Comment (mtrojan-ub) [inline SageJournals.js:2]: This translator will cause a conflict when merging, similar translators "SAGE Journals.js" with same guid + label exists

**Pull request closed without merging**